src/YesSql.Provider.SqlServer/SqlServerDialect.cs

```diff
@@ -206,9 +209,17 @@ public override string QuoteForColumnName(string columnName)
             return "[" + columnName + "]";
         }
 
-        public override string QuoteForTableName(string tableName)
+        public override string QuoteForTableName(string tableName, bool IncludeSchema = false)
         {
-            return "[" + tableName + "]";
+            var result = "[" + tableName + "]";
```

Comment:
How is it different from `QuoteForColumn` when `includeSchema` is false? I even question semantically. Meaning that it could be the same but we'd want to keep them with different methods, but here I don't think it's "a table without the schema" we want to render.
I will double check with your previous examples.

Reply:
Yes, that's why I had used QuoteForColumn because there was no difference. But not appropriate.